INSTALL.md

```diff
@@ -0,0 +1,25 @@
+
+## Setup CactusBot:
+```
+git clone https://github.com/cactusdev/cactusbot
+cd cactusbot
+pip3 install -r requirements.txt
```

Comment:
Use `pip`/`python`, not `pip3`/`python3` - assume that people have Python 3 as their main version.